src/library.js

```diff
@@ -1766,19 +1766,27 @@ LibraryManager.library = {
   // setjmp.h
   // ==========================================================================
 
-#if SUPPORT_LONGJMP
   longjmp__sig: 'vii',
+#if SUPPORT_LONGJMP
   longjmp: function(env, value) {
     _setThrew(env, value || 1);
     throw 'longjmp';
   },
+#else
+  longjmp__deps: [function() {
+    error('longjmp support was disabled (SUPPORT_LONGJMP=0), but it is required by the code (either set SUPPORT_LONGJMP=0, or remove uses of it in the project)');
```

Comment:
The advice there should be to remove `SUPPORT_LONGJMP=0` right?  Since the default is `1`

Reply:
Right, thanks, I was off by 1...